libs/barista-components/slider/index.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export { DtSlider } from './src/slider';
```

Comment:
```suggestion
export { DtSlider, DtSliderUnit, DtSliderLabel } from './src/slider';
```
You need to export all directives here as well. Others might want to grab the directive with viewchildren and need the type for that :)

Reply:
I'll add the exports, thanks.